libs/SerialNumberKit/include/SerialNumberKit.h

```diff
@@ -5,21 +5,28 @@
 #pragma once
 
 #include <array>
+#include <filesystem>
 #include <interface/drivers/MCU.h>
+#include <span>
 
 namespace leka {
 
 class SerialNumberKit
 {
-	static constexpr size_t SN_SIZE		  = 33;
-	static constexpr size_t SHORT_SN_SIZE = 16 + 1;
+	static constexpr size_t SN_SIZE			= 31 + 1;
+	static constexpr size_t SHORT_SN_SIZE	= 15 + 1;
+	static constexpr auto date_of_test_path = "fs/sys/date_of_test";
```

Comment:
declared here, they are all implicitly private, can we move them to the private part of the class then?

Reply:
Yes they are in implicit `private`, I just put them at the top since they are kind of "configuration" and easily accessible